components/eamxx/src/share/property_checks/mass_and_energy_column_conservation_check.cpp

```diff
@@ -362,4 +528,17 @@ compute_energy_boundary_flux_on_column (const Real vapor_flux,
   return vapor_flux*(LatVap+LatIce) - (water_flux-ice_flux)*RHO_H2O*LatIce + heat_flux;
 }
 
+Real MassAndEnergyColumnConservationCheck::get_echeck(){
```

Comment:
You can mark this and several other methods as `const`: `:get_echeck() const {`

Reply:
thanks, done